config/config.go

```diff
@@ -56,6 +56,8 @@ type MetricsConfig struct {
 	// Include node metadata such as 'node.kubernetes.io/instance-type' or
 	// 'topology.kubernetes.io/region'
 	IncludeNodeLabels bool `yaml:"includeNodeLabels"`
+	// Include node info such as kubelet version and kernel version.
+	IncludeNodeInfo bool `yaml:"includeNodeInfo"`
```

Comment:
Can we also take this opportunity to add a new config test scenario for metrics?  It looks like we don't have one that hits this struct.